src/unzip.spec.ts

```diff
@@ -13,7 +13,7 @@ describe("unzip", () => {
   });
 
   after(async () => {
-    await fs.promises.rmdir(tempDir, { recursive: true });
+    await fs.promises.rm(tempDir, { recursive: true });
```

Comment:
just checking that this is intentional :)

Reply:
Yep, `rmdir` recursive was deprecated (and the deprecation notice says to use this instead.